dgsudharsan/sonic-buildimage
# Removing execute permission from copp service file

#### Why I did it
Removed execute permissions from the systemd copp-config.service file. Without this we will get a warning
Aug 25 06:36:43.738669 r-anaconda-51 WARNING systemd[1]: Configuration file /lib/systemd/system/copp-config.service is marked executable. Please remove executable permission bits. Proceeding anyway.


#### How I did it
Changed file permissions

#### How to verify it
find files/image_config/ -name "*".service | xargs ls -l
-rw-r--r-- 1 sudharsand dip 196 May  4 03:06 files/image_config/copp/copp-config.service
-rw-r--r-- 1 sudharsand dip 231 May  4 03:06 files/image_config/corefile_uploader/core_uploader.service
-rw-r--r-- 1 sudharsand dip 135 May  4 03:06 files/image_config/fstrim/fstrim.service
-rw-r--r-- 1 sudharsand dip 263 May  4 03:06 files/image_config/hostname/hostname-config.service
-rw-r--r-- 1 sudharsand dip 263 May  4 03:06 files/image_config/interfaces/interfaces-config.service
-rw-r--r-- 1 sudharsand dip 292 May  4 03:06 files/image_config/ntp/ntp-config.service
-rw-r--r-- 1 sudharsand dip 418 May  4 03:06 files/image_config/ntp/ntp.service
-rw-r--r-- 1 sudharsand dip 216 Jun 16 21:31 files/image_config/pcie-check/pcie-check.service
-rw-r--r-- 1 sudharsand dip 257 May  4 03:06 files/image_config/rsyslog/rsyslog-config.service
-rw-r--r-- 1 sudharsand dip 233 May  4 03:06 files/image_config/system-health/system-health.service
-rw-r--r-- 1 sudharsand dip 283 May  4 03:06 files/image_config/topology/topology.service
-rw-r--r-- 1 sudharsand dip 229 May  4 03:06 files/image_config/warmboot-finalizer/warmboot-finalizer.service
-rw-r--r-- 1 sudharsand dip 169 May  4 03:06 files/image_config/watchdog-control/watchdog-control.service


#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog



#### A picture of a cute animal (not mandatory but encouraged)
